react/src/components/ForceGraph/ForceGraph.js

```diff
@@ -433,8 +433,50 @@ const ForceGraph = ({
   const handlePopupClose = () => setPopup({ ...popup, visible: false });
   const toggleOptionsVisibility = () => setOptionsVisible(!optionsVisible);
 
-  // Exports current SVG view as an image file (SVG or PNG).
+  // Exports the current view as an image file (SVG, PNG) or data file (JSON).
   const exportGraph = (format) => {
+    // Create string from originNodeIds, with default fallback.
```

Comment:
I did not expect this approach, but I get it. You need to create the object to download on the client, then trigger the download.